app/views/reports/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<% title "#{'New Report'}" %>
+<%= form_for(:search, :url => {:action => "create", :ally_id => params[:ally_id], :comment_id => params[:comment_id]}, :method => "post") do |f| %>   
+    <div class="table">
```

Comment:
I recently got rid of the table and field classes. So you don't need them and also the div with the clear class. If you need spacing between blocks, use `smallMarginBottom` or `smallMarginTop` depending on your needs.